### DISCUSSION TITLE
Future of OrientDB Enterprise edition

### DISCUSSION THREAD

**Discussion by milan-cvejic** [orientechnologies/orientdb#9538] [General] [answered]
SAP decided to remove OrientDB from their price list, and final release of Enterprise OrientDB will be 3.2, after that they will go to maintenance mode.

Is there any plan to include enterprise features in community edition?

- Non-Stop Backup and Restore
- Scheduled FULL and Incremental Backups
- Query Profiler
- Distributed Clustering configuration
- Metrics Recording
- Live Monitoring with configurable Alerts

Thanks

**Comment by misu200** [comment]
Is there any official announcement about this?
Should we worry about the future of OrientDB community edition?

**Comment by luigidellaquila** [comment, marked as answer]
Hi all,

This is a statement I can share, based on my understanding as a core OrientDB developer

```
“OrientDB Community Edition continuing to evolve as open-source project”
- With the envisioned product scope in reach, SAP Enterprise OrientDB 3.2 will be the final release 
  of the commercial product with a planned delivery in Q2/2021 and with continued support of existing 
  SAP Enterprise OrientDB customers.
- In addition, the OrientDB Community Edition will continue to evolve as an open-source project – contributions 
  by the community are continuing to be much valued.
```

About the release of Enterprise features in the CE, there is no decision in that sense as far as I know, but if it happens I'll be more than happy to let you know.


Thanks and keep going with OrientDB, the first Multi-Model database in the market; as a community member (even before than a committer and a core team member) I still see a lot of potential ahead; of course a big impact on this will also come from the involvement of the Community, so feel free to contribute, propose improvements, send pull-requests, and open/comment/fix issues of course ;-) 

Luigi
